modular_skyrat/modules/reagent_forging/code/reagent_component.dm

```diff
@@ -88,10 +88,14 @@
 
 /datum/component/reagent_weapon/proc/inject_attacked(datum/source, mob/living/target, mob/living/user, params)
 	SIGNAL_HANDLER
+	//don't have the weapon or any imbued reagents? don't try
 	if(!parent_weapon || !length(imbued_reagent))
 		return
+	//lets inject that target
 	var/mob/living_target = target
 	for(var/create_reagent in imbued_reagent)
 		living_target.reagents.add_reagent(create_reagent, 1)
```

Comment:
Same here, the `1` should be a define like `#define REAGENT_WEAPON_UNITS_PER_HIT 1` or something like that.